graylog2-server/src/main/java/org/graylog2/shared/inputs/PersistedInputs.java

```diff
@@ -23,6 +23,4 @@
     boolean add(MessageInput e);
     boolean remove(Object o);
     boolean update(String id, MessageInput newInput);
```

Comment:
If I'm not mistaken, the `add`, `remove` and `update` methods are not used in the code base. I think we should delete them from this interface and the implementation.

Reply:
You're right. I removed those methods.